src/lib/telldus-accessory.js

```diff
@@ -164,15 +167,48 @@ class TelldusDimmer extends TelldusSwitch {
    */
   getDimLevel(callback, context) {
     this.log('getDimLevel called')
+    
     TDtool.device(this.id).then(device => {
       if (device.dimlevel)
```

Comment:
You'd need braces around this when introducing this change; line 175 will be unreachable otherwise.